### ISSUE TITLE
(3.2.15, CompnentV2 build)`RwResolvedProjectV2Test >> testCreateNewDiskProject` gois into infinite loop

### ISSUE THREAD

**Issue by dalehenrich** [GemTalk/Rowan#561]
[More details here](https://github.com/GemTalk/Rowan/issues/560#issuecomment-570747120) ... suffice to say the the loop reproduces in a Jadeite interactive run of `RwResolvedProjectV2Test >> testCreateNewDiskProject` ... does not hit infinite loop in 3.5.0 ... likely a FileSystem issue specific to 3.2.15 ...

**Comment by dalehenrich** [comment]
Turns out that #keywords in 3.2.15 is wrong:
```
#test keywords anArray( 'test:')
```

**Comment by dalehenrich** [comment]
Started using the tonel-gemstone method Symbol>>_keywords which gave right answer for #test, but below is another infinite loop in 3.2.15:
```
ERROR 6003 , a Break occurred (error 6003), A soft break was received. nil (Break)
topaz > exec iferr 1 : stk 
==> 1 Break (AbstractException) >> _signalWith:     @5 line 25
2 [] in  ExecBlock0 (AbstractException) >> _signalAsync @2 line 19
3 ExecBlock0 (ExecBlock) >> ensure:             @2 line 12
4 Break (AbstractException) >> _signalAsync     @6 line 21
5 Array (Collection) >> do:                     @2 line 9
6 TonelWriter >> selectorIsComplete:in:         @3 line 5
7 TonelWriter >> splitMethodSource:into:        @21 line 12
8 TonelWriterForCypressTest (TonelWriterTest) >> testSplitMethodSourceInto @14 line 14
9 TonelWriterForCypressTest (TestCase) >> performTest @3 line 2
10 [] in  ExecBlock0 (TestCase) >> runCase       @3 line 5
11 ExecBlock0 (ExecBlock) >> ensure:             @2 line 12
12 TonelWriterForCypressTest (TestCase) >> runCase @4 line 6
13 [] in  ExecBlock0 (TestResult) >> runCase:    @2 line 2
14 ExecBlock0 (ExecBlock) >> on:do:              @3 line 42
15 TestResult >> runCase:                        @7 line 4
16 TonelWriterForCypressTest (TestCase) >> run:  @2 line 2
17 [] in  ExecBlock1 (TestSuite) >> run:         @3 line 4
18 OrderedCollection (Collection) >> do:         @5 line 10
19 TestSuite >> run:                             @3 line 2
20 [] in  ExecBlock0 (TestSuite) >> run          @2 line 4
21 ExecBlock0 (ExecBlock) >> ensure:             @2 line 12
22 TestSuite >> run                              @4 line 5
23 [] in  Executed Code                          @2 line 78
24 ExecBlock0 (ExecBlock) >> on:do:              @3 line 42
25 [] in  Executed Code                          @30 line 79
26 ExecBlock0 (ExecBlock) >> ensure:             @2 line 12
27 Executed Code                                 @10 line 137
28 GsNMethod class >> _gsReturnToC               @1 line 1
```

**Comment by dalehenrich** [comment]
additional 3.2.15 regresssions addressed with commit 25206664
